clickhouse-admin/src/http_entrypoints.rs

```diff
@@ -21,11 +23,29 @@ enum ClickhouseAdminImpl {}
 impl ClickhouseAdminApi for ClickhouseAdminImpl {
     type Context = Arc<ServerContext>;
 
-    async fn clickhouse_address(
+    async fn generate_server_config(
         rqctx: RequestContext<Self::Context>,
-    ) -> Result<HttpResponseOk<ClickhouseAddress>, HttpError> {
+        path: Path<GenerationNum>,
+        body: TypedBody<ServerSettings>,
+    ) -> Result<HttpResponseCreated<ReplicaConfig>, HttpError> {
         let ctx = rqctx.context();
-        let output = ctx.clickward().clickhouse_address()?;
-        Ok(HttpResponseOk(output))
+        let server_settings = body.into_inner();
+        let output = ctx.clickward().generate_server_config(server_settings)?;
+        // TODO(https://github.com/oxidecomputer/omicron/issues/5999): Do something with the generation number
+        println!("{path:?}");
+        Ok(HttpResponseCreated(output))
+    }
+
+    async fn generate_keeper_config(
+        rqctx: RequestContext<Self::Context>,
+        path: Path<GenerationNum>,
+        body: TypedBody<KeeperSettings>,
+    ) -> Result<HttpResponseCreated<KeeperConfig>, HttpError> {
+        let ctx = rqctx.context();
+        let keeper_settings = body.into_inner();
+        let output = ctx.clickward().generate_keeper_config(keeper_settings)?;
+        // TODO(https://github.com/oxidecomputer/omicron/issues/5999): Do something with the generation number
+        println!("{path:?}");
```

Comment:
Log this instead?

Reply:
Whoops! I meant to get rid of this altogether 😅